Formula/skktools.rb

```diff
@@ -24,4 +24,50 @@ def install
     ENV.deparallelize
     system "make", "install"
   end
+
+  test do
+    (testpath/"SKK-JISYO.TEST").write <<~EOS.strip.tap { |s| s.encode("euc-jis-2004") }
+      わるs /悪/
+      わるk /悪/
+      わるi /悪/
+    EOS
+
+    (testpath/"SKK-JISYO.SHUFFLE").write <<~EOS.tap { |s| s.encode("euc-jis-2004") }
+      わるs /悪/
+      わるi /悪/
+      わるk /悪/
+    EOS
+
+    expect_shuffle = <<~EOS.strip.tap { |s| s.encode("euc-jis-2004") }
+      ;; okuri-ari entries.
+      わるs /悪/
+      わるk /悪/
+      わるi /悪/
+    EOS
+
+    (testpath/"SKK-JISYO.SPLIT1").write <<~EOS.strip.tap { |s| s.encode("euc-jis-2004") }
+      わるs /悪/
+      わるk /悪/
+    EOS
+
+    (testpath/"SKK-JISYO.SPLIT2").write <<~EOS.strip.tap { |s| s.encode("euc-jis-2004") }
+      わるk /悪/
+      わるi /悪/
+    EOS
+
+    (testpath/"SKK-JISYO.SPLIT3").write <<~EOS.strip.tap { |s| s.encode("euc-jis-2004") }
+      わるi /悪/
+    EOS
+
+    expect_expr = <<~EOS.strip.tap { |s| s.encode("euc-jis-2004") }
+      ;; okuri-ari entries.
+      わるs /悪/
+      わるk /悪/
+    EOS
+
+    assert_equal "SKK-JISYO.TEST: 3 candidates", shell_output("#{bin}/skkdic-count SKK-JISYO.TEST").strip
+    assert_equal expect_shuffle, shell_output("cat SKK-JISYO.SHUFFLE | #{bin}/skkdic-sort").strip
+    assert_equal expect_expr, shell_output("#{bin}/skkdic-expr SKK-JISYO.SPLIT1 + SKK-JISYO.SPLIT2 - SKK-JISYO.SPLIT3 | #{bin}/skkdic-sort").strip
```

Comment:
You can use `pipe_output` for these rather than `shell_output`. We want to wrap to 80 characters as well.

Reply:
Hi @DomT4 

Thank you for review.
I fixed it to wrap to 80 characters.

Is it ok? or I misunderstand it?


Thanks.